README.md

```diff
@@ -14,9 +14,7 @@
 [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-908a85?logo=gitpod)](https://gitpod.io/from-referrer/)
 # AI assistant for software developers
 
-> :bulb: **Note:**
```

Comment:
If you will copy-paste the actual emoji and not the : --- : it should work, as it a valid Unicode.

Reply:
Removed this anyway,  thanks